### ISSUE TITLE
what happens when user clicks remind me later?

### ISSUE THREAD

**Issue by talbussel** [pushandplay/cordova-plugin-apprate#134]
Great Plugin! saved me a lot of problems trying to open Google Play URL from native app :-)

I guess from the checks i did that nothing happen  when user clicks remind me later, and each developer should decide what to do, am i wrong?

If i'm right - could be very cool to define a var like "usesUntilPromptAgain"

Cheers!
Tal

**Comment by smeeguel** [comment]
Agreed with Tal! My quick 'n' dirty solution is to clear the counter in local storage:

`      this.appRate.preferences.callbacks = 
      {
        onRateDialogShow: () =>
        {
          console.log("rate dialog show");
        },
        onButtonClicked: (value) =>
        {
          console.log("clicked button " + value);
          switch (value) {
            // 1 = write review
            case 1:
              console.log("rate dialog review");
              break;
            // 2 = remind me later
            case 2:
              console.log("rate dialog later");
              // Clear the counter so app asks again later
              window.localStorage.removeItem('counter');
              break;
            // 3 = no thanks
            case 3:
            default:
              console.log("rate dialog no thanks");
              break;
          }
        }
      }
`

**Comment by 7h3m4dS0u1** [comment]
How it's today using local storage... like @smeeguel solution.
(...)
`promptForRatingWindowButtonClickHandler = function(buttonIndex) {
    var base;
    switch (buttonIndex) {
      case 1:
        updateCounter('stop');
        AppRate.navigateToAppStore();
        break;
      case 2:
        updateCounter('reset');
        break;
      case 3:
        updateCounter('stop');
    }
    return typeof (base = AppRate.preferences.callbacks).onButtonClicked === "function" ? base.onButtonClicked(buttonIndex) : void 0;
  };`
(...)
